### DISCUSSION TITLE
How to create expiry date input and card no. input using Tailwind and Functional component approach ?

### DISCUSSION THREAD

**Discussion by ProPhycient5** [tailwindlabs/tailwindcss#10085] [Help] [answered]
How can I create an expiry date input as a functional component?

This type of input, we find while doing online card payments. Like the below one, once you type 2 characters, `/` will appear and it will accept other 2 characters(number)
![input_1](https://user-images.githubusercontent.com/71059909/207780751-5349df2b-97fc-4393-80ff-498082e1f683.JPG)

**Comment by wongjn** [comment, marked as answer]
This is out of scope of Tailwind. You'd need to use JavaScript to achieve your desired behavior.

**Reply by ProPhycient5** [reply]
Okay @wongjn
